eva_submission/eload_utils.py

```diff
@@ -262,3 +263,14 @@ def create_assembly_report_from_fasta(assembly_fasta_path):
             open_file.write('\t'.join(['na', 'na', 'na', 'na', contig_name, '<>', 'na', 'na', str(seq_length), 'na']) + '\n')
     return assembly_report_path
 
+
+def get_nextflow_config_flag():
+    """
+    Return the commandline flag for Nextflow to use the config provided in environment variable NEXTFLOWCONFIG.
+    If not provided, return an empty string, which allows Nextflow to use the default precedence as described here:
+    https://www.nextflow.io/docs/latest/config.html
+    """
+    env_val = os.getenv('NEXTFLOWCONFIG')
```

Comment:
We should coordinate to use the same environment variable for this.
I've used `RELEASE_NEXTFLOW_CONFIG` but happy to change it.

Reply:
I guess being process-specific makes sense, can do `SUBMISSION_NEXTFLOW_CONFIG`